nms/app/packages/magmalte/app/components/context/RefreshContext.js

```diff
@@ -117,6 +124,8 @@ export function useRefreshingContext(props: Props) {
       return ctx.setState(null, null, null, newState);
     } else if (props.type === RefreshTypeEnum.FEG_GATEWAY) {
       return ctx.setState(null, null, newState?.fegGateways || {});
+    } else if (props.type === RefreshTypeEnum.FEG_SUBSCRIBER) {
+      return ctx?.setSessionState(newState);
```

Comment:
Why is optional chaining used here?

Reply:
Oh, I was being extra protective so as to not get flow error because ctx doesn't contain the setSessionState function for other types like enodebs or gateways but it turns out flow doesn't complain about that. So, I have removed it in order to make it similar to the other code.